appcues/src/main/java/com/appcues/ui/AppcuesViewModel.kt

```diff
@@ -84,13 +112,40 @@ internal class AppcuesViewModel(
         }
     }
 
+    fun onPresentationComplete() {
+        uiState.value.let { state ->
+            if (state is Rendering) {
+                appcuesCoroutineScope.launch {
+                    state.presentationComplete(Success(Unit))
+                }
+            }
+        }
+    }
```

Comment:
this is called from the `AppcuesComposition` now, to make usage in that spot simpler and encapsulate the usage of the `appcuesCoroutineScope` here.